blogs/2022-06-28-tips-kafka-streams-distributed.md

```diff
@@ -11,18 +11,18 @@ image: /blogs/2022-06-28-tips-kafka-streams-distributed.jpg
 draft: true
 ---
 
-While building [Kestra](https://github.com/kestra-io/kestra), an open-source data orchestration and scheduling platform, we decided to use Kafka as the main database to build a scalable architecture. Since Kafka allows gigabytes of throughput, we are confident to have a fully scalable solution, but we need to adapt to Kafka.
+When it comes to advantages, the destributed event store and streamlining platform Kafka has many. Think high-throuput, low latency, durability, high concurancy, scalability, and fault-tolerant, and you have an idea of what I am talking about. So, its not suprising that when we built [Kestra](https://github.com/kestra-io/kestra), an open-source data orchestration and scheduling platform, we decided to use Kafka as the main database to build a scalable architecture. Since Kafka allows gigabytes of throughput, which allows it to handle high-volume data at equally high velocity, we're confident of a fully scalable solution. However, we need to adapt to Kafka.
 
-Since we heavily rely on [Kafka Streams](https://kafka.apache.org/documentation/streams/) for most of our services (executor and scheduler mostly), we have made some assumptions on how it handles the workload. This blog post is here to show some advanced techniques we have to use to have a Kafka Stream reliable, we want to share with you the tips we have discovered during the last two years.
+Considering our heavy reliance on [Kafka Streams](https://kafka.apache.org/documentation/streams/) for most of our services (executor and scheduler in our case), we have made some assumptions regarding how it handles the workload. In this blog post, my aim is to show some advanced techniques employ to make Kafka Stream more reliable. I'll share some tips we have discovered during the last two years.
 
 ## Same topic as source and destination
```

Comment:
I think after each heading, you may need a sentence or two giving the context of the heading. For example, in this sub-header, you may start by explaining how the "Same topic as a source and destination" relates to the main topic. What does this mean in the context of this article?